sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -538,39 +538,6 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       sql("SELECT 1, 2, sum(b) FROM testData2"))
   }
 
-  test("Group By Ordinal - negative cases") {
```

Comment:
hmmm, I know they will be tested in #14595 soon, but after we merge this PR and before merge #14595, this feature will be untested. Should we allow this? cc @rxin